src/client/components/AddDirectMessageModal.tsx

```diff
@@ -199,6 +201,7 @@ const Avatar = styled(DefaultAvatar)`
 const Heading = styled.h2({
   display: 'flex',
   marginTop: 0,
+  marginBottom: 0,
```

Comment:
⛏️ You could use the neat shorthand.
```suggestion
  marginBlock: 0,
```

Reply:
TIL!